src/common.rs

```diff
@@ -19,6 +19,7 @@ pub enum Architecture {
     #[allow(non_camel_case_types)]
     X86_64_X32,
     Hexagon,
+    HPPA,
```

Comment:
```suggestion
    Hppa,
```
This is the usual rust naming convention for acronyms, which we also use for other variants of this enum.

Reply:
I can certainly do that although it's a bit odd since "HPPA" stands for "Hewlett Packard Precision Architecture" with "Hppa" being "Hewlett packard precision architecture". On the other hand, we also have "Sparc" instead of the correct "SPARC".

But as you wish ;-).

Reply:
OK, done.